docs/UserGuide.md

```diff
@@ -36,99 +36,143 @@
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+  `done 2`<br>
+  Mark the 2nd event or task in the task manager as done.
 
 #### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
+Clears all entries from the calendar.<br>
 Format: `clear`  
 
 #### Exiting the program : `exit`
 Exits the program.<br>
 Format: `exit`  
 
-#### Saving the data 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
+#### Saving the data: 
+Data will be automatically save to the default location after any command that changes the data.<br>
+If user has changed the file location using the ‘change’ command, data will be save to the file path that user indicate.<br>
 There is no need to save manually.
 
+####Changing default storage location: `change`
+Default storage location will be changed to the location specified by the user. Any data saved in the previous location will be cleared if specified.<br>
+Format: `change FILE_PATH`
+
+> The file path must end with the file type extension, .xml
+
+Example: `change /Desktop/folder/taskManager.xml clear`
+
+#### Undo operations: `undo`
+Undo the most recent action.<br>
+Format: `undo` 
+
+#### Filter events and tasks: `filter`
+Filter list for attributes such as event or task, done or yet to be done, start time, deadline, tag, and priority level.<br>
+Format: `filter [n/EVENT/TASK] [s/START_DATE] [d/DEADLINE] [c/DONE/UNDONE] [t/TAG] [p/PRIORITY_LEVEL]` 
+
+Examples:
+* `filter n/event s/7.10.2016-14 t/CS2103 `
+  List events that start from 7.10.2016-14 and have tag CS2103.
+* `filter n/task c/undone p/3 `
+  List tasks that are yet to be done and have priority 3.
+
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with 
-       the file that contains the data of your previous Address Book folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with
+       the file that contains the data of your previous TaskManager folder.
        
 ## Command Summary
 
 Command | Format  
--------- | :-------- 
-Add | `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+-------- | :--------
+Add | `add EVENT_NAME s/START_DATE e/END_DATE [t/TAG] [p/PRIORITY_LEVEL]`<br> `add TASK_NAME [d/DEADLINE] [t/TAG] [p/PRIORITY_LEVEL]`
+Change | `change FILEPATH`
 Clear | `clear`
-Delete | `delete INDEX`
+Delete | `delete INDEX`<br> `delete EVENT_NAME/TASK_NAME`
+Done | `done INDEX`
+Edit | `edit TASK_NAME [d/DEADLINE] [t/TAG] [p/PRIORITY_LEVEL]`<br> `edit EVENT_NAME s/START_DATE e/END_DATE [t/TAG] [p/PRIORITY_LEVEL]`
+Filter | `[n/EVENT/TASK] [s/START_DATE] [d/DEADLINE] [c/DONE/UNDONE] [t/TAG]`
 Find | `find KEYWORD [MORE_KEYWORDS]`
-List | `list`
 Help | `help`
-Select | `select INDEX`
+List | `list`
+Undo | `undo`
```

Comment:
The 'Command summary' section will be used heavily by our testers. Try to make it easy to use as possible. e.g. give examples for each command.